app/domain/logs.rb

```diff
@@ -121,7 +121,7 @@ module ResourceRestrictions
 
       ExtractingRestrictionsFromResource = ::Util::TrackableLogMessageClass.new(
         msg: "Extracting resource restrictions for authenticator '{0-authn-name}' from host '{1-host-name}'",
-        code: "CONJ00039D"
+        code: "CONJ00032D"
```

Comment:
Is this one of our (CyberArk) authenticators?
Can you give me an example of filled in....


If one of ours, then,  

Extracting resource restrictions for {0-authn-name} Authenticator from host '{1-host-name}'",
for example: 
"Extracting resource restrictions for Azure Authenticator from host '{1-host-name}'",